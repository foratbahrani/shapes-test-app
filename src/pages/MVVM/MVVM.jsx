import React, { Component } from 'react';

export default class MVVM extends Component {
	render() {
		return <div>Second Solution without the use of redux (to be done)</div>;
	}
}
