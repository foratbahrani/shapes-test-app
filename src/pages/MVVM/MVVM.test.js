import { shallow } from 'enzyme';
import React from 'react';
import MVVM from './MVVM';

it('matches snapshot', () => {
	expect(shallow(<MVVM />)).toMatchSnapshot();
});
