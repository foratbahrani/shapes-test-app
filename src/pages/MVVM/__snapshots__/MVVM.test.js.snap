// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  Second Solution without the use of redux (to be done)
</div>
`;
